README.md

```diff
@@ -111,11 +111,23 @@ following types of records from NSF:
     - grants
     - researcher_funds
 
-1. **Open Access Button** - We import URLs to the content of open access publications that are provided by
+1. **Open Access Button** - We import information about open access copies of publications that is provided by
 Open Access Button via their web [API](https://openaccessbutton.org/api). We only look up publications in
-Open Access Button by DOI, so this import only adds data to existing publications in our database that have
+Open Access Button by DOI, so this import only creates records for existing publications in our database that have
 DOIs. A cron job automatically runs this import in production once per week beginning at around 8:00 AM on
-Sunday. This import can sometimes take more than a day to finish.
+Sunday. This import can sometimes take more than a day to finish. We import the following types of records from
```

Comment:
Since we expanded the scope of which publications we import data for (now considering all publications with a DOI, regardless of open access status), I think the OAB import now can take multiple days to finish as well.